lib/Auth/Process/ForceAup.php

```diff
@@ -304,4 +306,26 @@ private function mergeAupsToBeApproved(array $perunAupsToBeApproved, array $voAu
         }
         return $resultAups;
     }
+
+    /**
+     * Parses datetime with format set in self::DATETIME_FORMAT. If parsing fails, value passed in $default will be
+     * returned, if not null. Otherwise, January 1st, 1970 will be returned.
+     *
+     * @param string $date to be parsed using self::DATETIME_FORMAT format
+     * @param DateTime|null $default (optional) value to be returned in case of error,
+     * otherwise, January 1st, 1970 is returned in case of parsing error.
+     * @return DateTime parsed datetime, or default value in case of error (January 1st, 1970 if not provided)
+     */
+    private function parseDateTime(string $date, DateTime $default = null): DateTime
+    {
+        $result = DateTime::createFromFormat(self::DATETIME_FORMAT, $date);
+        if ($result === false) {
+            if ($default !== null) {
+                $result = $default;
+            } else {
+                $result = (new DateTime())->setTimestamp(0);
+            }
+        }
```

Comment:
this changes behavior a little bit, so 9109199c7c4c1c1f5065db4ce013d90d2f693d34 should be a `fix:`

Reply:
Changed to fix